crates/rspack_plugin_javascript/src/visitors/inject_runtime_helper.rs

```diff
@@ -81,6 +81,9 @@ impl<'a> VisitMut for InjectRuntimeHelper<'a> {
         return;
       }
 
+      if "_instanceof" == word {
```

Comment:
Because we can't shake the `_instanceof` for now, also there is nothing to add to `runtime_requirements`, ignore it to avoid panic